### PR TITLE
Backdrop is on top of overlay in iOS devices

### PR DESCRIPTION
**Pull request by avizcaino**
Hi,

I found a bug in iOS devices, where the backdrop is positioned on top of the overlay, avoiding to select the paper-fab-speed-dial-actions below.

This is discussed here: https://github.com/PolymerElements/iron-overlay-behavior/issues/208

> This is due to how stacking context works: any element that creates a new stacking context will "trap" all its children within it (overlays included), try it yourself.

It seems that the backdrop is added later to the DOM, causing the z-index assigned to the `paper-fab-speed-dial` element to be ignored.

I created a work-around where it is possible to assign a `withBackdrop` attribute to the `paper-fab-speed-dial` element such as:

```
<paper-fab-speed-dial with-backdrop.bind="true">
...
</paper-fab-speed-dial>
```

Additionally, since this does not solve the problem with the iOS devices, I added a second check where the userAgent is tested and if the browser is Safari Mobile, the `withBackdrop` attribute value is overridden and assigned to false:

```
_withBackdropChanged: function(withBackdrop) {
    if(withBackdrop){
        /**
        * Due to incompatibility with the backdrop showing on top of the overlay
        * in iOS devices, override the value of withBackdrop property to hide it.
        */
        var userAgent = window.navigator.userAgent;
        if (userAgent.match(/iPad/i) || userAgent.match(/iPhone/i)) {
            // iPad or iPhone
            this.withBackdrop = false;
        }
    }
}
```

Hope you find this useful.

### CODE REVIEW

File: paper-fab-speed-dial.html [hunk 1]
```diff
@@ -66,36 +66,60 @@
...
-				notify: true
-			}
-		},
-
-		// Public methods
-		open: function(e) {
-			// Required for mobile Safari to avoid passing the tap event to an element below the FAB
-			if (e) {
-				e.preventDefault();
+	(function() {
```
Comment (ronnyroeller) [inline paper-fab-speed-dial.html]: Could you please remove the tab intend, so the diff of your PR contains only code that reverse to the backdrop issue?

**Comment by avizcaino** [comment]
Done! Sorry about that.

**Comment by ronnyroeller** [comment]
Thanks a lot!

**Comment by ronnyroeller** [comment]
@avizcaino I re-tested the original version (version 1.0.12) with the latest Polymer version and Polymer element dependencies on Safari of a recent iOS version (9.3.5). This seems to work now well with the backdrop. I'd therefore remove again the backdrop disabling if possible (as it looks better on iOS).
Could you double check if the latest setup works also for you without the backdrop disabling? The original version is `paper-fab-speed-dial#1.0.12`.

**Comment by avizcaino** [comment]
Hey! Sorry, I didn't check yet if this is solved in iOS 9.3.5... I'll tell you something as soon as I can.

**Pull request merged**